File: lab4/s1/ejb/src/main/java/ru/uvuv643/ejb/remote/dto/human/response/DeleteResponseDto.java
package ru.uvuv643.ejb.remote.dto.human.response;

import ru.uvuv643.ejb.remote.dto.human.HumanBeingDto;

import  jakarta.xml.bind.annotation.XmlAccessType;
import  jakarta.xml.bind.annotation.XmlAccessorType;
import  jakarta.xml.bind.annotation.XmlElementWrapper;
import  jakarta.xml.bind.annotation.XmlRootElement;
import java.io.Serializable;
import java.util.List;

@XmlRootElement(name = "DeleteResponseDto")
@XmlAccessorType(XmlAccessType.FIELD)
public class DeleteResponseDto implements Serializable {

        @XmlElementWrapper(name = "teamList")
        private List<HumanBeingDto> teamList;

        public DeleteResponseDto() {
                this(null);
        }

        public DeleteResponseDto(List<HumanBeingDto> teamList) {
                this.teamList = teamList;
        }

        public List<HumanBeingDto> getTeamList() {
                return teamList;
        }

        public void setTeamList(List<HumanBeingDto> teamList) {
                this.teamList = teamList;
        }
}

File: lab1/s1/src/main/kotlin/ru/uvuv643/soa/api/v1/dto/human/response/DeleteResponseDto.kt
package ru.uvuv643.soa.api.v1.dto.human.response

import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty
import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement
import io.swagger.v3.oas.annotations.media.Schema
import ru.uvuv643.soa.api.v1.dto.human.HumanBeingDto

@JacksonXmlRootElement
@Schema(title = "addingHeroResponse", description = "Result of add-hero request")
data class DeleteResponseDto constructor(

        @field:Schema(
                description = "New Team List",
        )
        @param:JacksonXmlProperty(namespace = "teamList")
        val teamList: List<HumanBeingDto>?,

        )